tests/functional/aws-node-sdk/test/versioning/objectCopy.js

```diff
@@ -133,6 +134,45 @@ testing('Object Version Copy', () => {
             });
         }
 
+        it('should copy the appropriate version into the same bucket',
+        () => s3.putObjectAsync({ Bucket: sourceBucketName,
+              Key: sourceObjName,
+              Body: secondContent })
+            .then(() => s3.copyObjectAsync({
+                Bucket: sourceBucketName,
+                CopySource: `${sourceBucketName}/${sourceObjName}?` +
```

Comment:
minor: it should be the same as the `copySource` variable set in the beforeEach, right?